nasa/harmony
# HARMONY-1563: Default to ignoreErrors=true behavior to continue processing requests on individual failures

## Jira Issue ID
HARMONY-1563

## Description
Changes our default behavior from failing requests on the first work item failure to continue processing.

## Local Test Steps

1. `npm run build-all-services` to make sure the work item updater task is built.
2. `bin/restart-services`
3. Submit the request: http://localhost:3000/C1234724470-POCLOUD/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?subset=lat(-45%3A45)&subset=lon(0%3A180)&skipPreview=true&granuleId=G1236546414-POCLOUD&granuleId=G1235845770-POCLOUD&granuleId=G1236614412-POCLOUD which has 2 failing and one successful granule. Verify that it completes with a COMPLETE_WITH_ERRORS state.
4. Submit another request with ignoreErrors=true and verify the behavior is the same (COMPLETE_WITH_ERRORS).
5. Submit another request with ignoreErrors=false and verify on the first failure it moves to the FAILED state and cancels the last work item instead of processing it.

## PR Acceptance Checklist
